packages/core/helpers/ver/4.24.9.ts

```diff
@@ -8,10 +9,28 @@ const patchSingleStateZoom = newConfig => {
   }
 }
 
+const patchDashboardFilterActiveStatus = newConfig => {
+  if (!newConfig.dashboard) return newConfig
+  const dashboardConfig = newConfig as MultiDashboardConfig
+
+  // previous filters need an active status
+  const newSharedFilters = (dashboardConfig.dashboard.sharedFilters || []).map(sf => {
+    if (!sf.active && sf.values) {
+      sf.active = sf?.values?.[0] || 'Select'
```

Comment:
values would be defined in this case. as would sf. 'Select' regardless shouldn't be the default. it should be just like null or empty string and you should have set a reset label.